specify/specify7
# Add a tool to select a record set in Accession

Fixes #2377 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- open a new accession form 
- [ ] verify there is no collection object present 
- [ ] verify that you can save accession without any CO 
- click the :mag_right: in the subview header next to the CO subtitle 
- click record set button 
- [ ] verify the records set are from the same table (CO) 
- choose one record set 
- click Add
- [ ] verify that the COs from the RS have been added to the COs subview on the accession form 
- [ ] verify that you can save the accession 
- open a new accession form 
- repeat all the steps from before but this time: 
- [ ] verify that you can add 2 record sets at the same time 

=> works for CO on CE, Appraisals on Accession, CO on Accession etc
